reference/5.1/Microsoft.PowerShell.Core/About/about_History.md

```diff
@@ -1,28 +1,25 @@
 ---
 keywords: powershell,cmdlet
 locale: en-us
-ms.date: 06/09/2017
+ms.date: 03/09/2020
 online version: https://docs.microsoft.com/powershell/module/microsoft.powershell.core/about/about_history?view=powershell-5.1&WT.mc_id=ps-gethelp
 schema: 2.0.0
 title: about_History
 ---
-
 # About History
 
 ## Short Description
-
 Describes how to get and run commands in the command history.
 
 ## Long Description
 
-When you enter a command at the command prompt, Windows PowerShell saves the
-command in the command history. You can use the commands in the history as a
-record of your work. And, you can recall and run the commands from the command
-history.
+When you enter a command at the command prompt, PowerShell saves the command in
+the command history. You can use the commands in the history as a record of
+your work. And, you can recall and run the commands from the command history.
 
 ### History Cmdlets
 
-Windows PowerShell has a set of cmdlets that manage the command history.
```

Comment:
This is in the 5.1 doc, Should  "Windows" be deleted?

Reply:
Yes, because the information is not Windows 5.1 specific. We keep "Windows PowerShell" when the feature only applies to Windows PowerShell.